app/assets/stylesheets/components/_navbar.scss

```diff
@@ -81,6 +81,13 @@
   li {
     padding: 3px 12px;
     color: $medium-gray;
+    display: flex;
+    align-items: center;
+    .nav-btn {
+      @extend .button-primary;
+      width: 144px;
```

Comment:
i made this `min-width`. Buttoms shouldn't have fixed widths, in case the user has a higher font size

Reply:
Nice trick. 
makes sense